mcs/class/System/System.Diagnostics/Process.cs

```diff
@@ -1253,65 +867,6 @@ public static Process Start(string fileName, string arguments, string username,
 		}
 #endif // MONO_FEATURE_PROCESS_START
 
-		public override string ToString()
-		{
-			return(base.ToString() + " (" + this.ProcessName + ")");
```

Comment:
this is fixed by the way we now treat `FEATURE_PAL` in referencesource `Process.cs